Final Project/Editor Interface/editor_interface_final.py

```diff
@@ -266,142 +284,132 @@ class ClickInfo(plugins.PluginBase):
         }                   
     }  
     """
-
+    
     def __init__(self, p, l):
         self.dict_ = {"type": "clickinfo",
                       "id": [utils.get_id(i) for i in p],
                       "labels": l
                       }
         print([utils.get_id(i) for i in p])
 
-
-def func2(ra, dec, mag, fov):
-    # sorting objects under the user input of limiting magnitude;
-    # also duplicating the the objects and transforming them so they are repeated to the left of y-axis
-    mag_ng = ng[(ng["V"] <= mag)]
```

Comment:
Can you check in the final database that is provided by Shyadari? I think it uses `V` instead of `mag`, that's why I think I changed this initially.

Reply:
That is correct. I checked in the new catalog of NGC.csv, the column names are changed. So I'll need to modify RA/DECs as well. Will make another commit.

Reply:
In the new commit, I have fixed all the column discrepancies. It should work now.